src/delegate/redux/selectors.js

```diff
@@ -127,4 +129,10 @@ const getFormattedDelegateRules = createSelector(
   },
 )
 
+export const getConnectedDelegateASTApproval = createSelector(
```

Comment:
@syjk129  Use this to get the state for if AST has been approved on the delegate contract